flask/json.py

```diff
@@ -222,8 +234,10 @@ def get_current_user():
             "id": 42
         }
 
-    For security reasons only objects are supported toplevel.  For more
-    information about this, have a look at :ref:`json-security`.
+
+    .. versionchanged:: 1.0
+       Added support for serializing top-level arrays. This introduces a
+       security risk in ancient browsers. See :ref:`json_security` for details.
```

Comment:
For some reason Sphinx isn't hyperlinking `json_security` on the generated docs. I'm not sure why. See the problem here: http://flask.pocoo.org/docs/dev/api/#flask.json.jsonify